.github/workflows/codeql.yml

```diff
@@ -29,5 +29,10 @@ jobs:
           languages: go
           queries: security-and-quality
 
+      - name: Setup go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22.1'
```

Comment:
Is there a reason you are pinning to `1.22.1` here rather than `1.22` everywhere else?